### DISCUSSION TITLE
Page Dynamic Table

### DISCUSSION THREAD

**Discussion by wsteixeira** [po-ui/po-angular#913] [Tire suas dúvidas] [answered]
A propriedade "optionsService" não está funcionando neste template "Page Dynamic Table", eu fiz assim: { property: 'peopleId', label: 'Nome', required: true, optionsService: '/api/people', fieldLabel: 'name', fieldValue: 'id', visible: false}, mas funcionou no template "Page Dynamic Edit".

**Comment by jhonyeduardo** [comment, marked as answer]
Bom dia @wsteixeira 

No PoPageDynamicTable os `fields` são utilizados para colunas e filtros (caso especificado), no caso o optionsService seria possível utilizado como filtro.

Pois o intuito do DynamicTable é listagem de dados, portanto não é possível utiliza-lo na tela principal, apenas na busca avançada (filtro)

**Reply by wsteixeira** [reply]
Obrigado!
